### ISSUE TITLE
Slitherin scan fails

### ISSUE THREAD

**Issue by spaceh3ad** [pessimistic-io/slitherin#106]
I installed slitherin using pip (python3.8).
My machine macos.

```
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.8/bin/slitherin", line 5, in <module>
    from slitherin.cli import main
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/slitherin/cli.py", line 9, in <module>
    import slither_pess
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/slither_pess/__init__.py", line 1, in <module>
    from slither_pess.detectors.arbitrary_call.arbitrary_call import ArbitraryCall
ModuleNotFoundError: No module named 'slither_pess.detectors.arbitrary_call'
```

**Comment by njelich-hacken** [comment]
The current public package has incorrect paths, fixed in the following commit:
https://github.com/pessimistic-io/slitherin/commit/687f842e1073856d9f56e7f0c59826c06cbabaeb

**Comment by ndkirillov** [comment]
Fixes merged with #108
